src/base/BaseScene.cc

```diff
@@ -1581,3 +1613,21 @@ void BaseScene::CreateMaterials()
   material->SetReceiveShadows(true);
   material->SetLightingEnabled(true);
 }
+
+/*
```

Comment:
Did you want to add another comment about uncommenting this code in gz-rendering8?

Reply:
oops, actually this code is no longer needed. Removed. [b4a27fa](https://github.com/gazebosim/gz-rendering/pull/845/commits/b4a27fa08173162565dde561203ccc9273bf8126)